addons/hr/hr.py

```diff
@@ -247,10 +247,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user = self.pool['res.users'].browse(cr, uid, user_id, context=context)
+            return {'value': {'name': user.name, 'work_email': user.email, 'image_medium': user.image_medium}}
```

Comment:
Thats what im saying, I checked XML file with all fields and it was using medium image. I could imagine that tere is some sort of possibility to set this image in return method, however I also spot that employees are never show big image, they are always displayed with medium.

Anyway, would you accept this PR? or i need to change something more?


Offtopic:
What is the story with hr.Holidays module? it seems it count weekend days as active days, so for 2 full weeks instead of counting it as 10 days, it count it as 12 days including weekends in between. that is wrong!

I sopt requests on forum:
https://www.odoo.com/forum/help-1/question/exclude-weekend-from-number-of-days-in-leave-requests-11083
or
https://www.odoo.com/forum/help-1/question/how-to-exclude-the-weekends-i-e-saturday-and-sunday-from-number-leave-1492

Seems like its not implemented since 2013, from odoo v6. Ill look at it, as we need it for our comoany